src/main/blackbox/blackbox.c

```diff
@@ -914,6 +914,39 @@ static bool sendFieldDefinition(const char * const *headerNames, unsigned int he
 }
 
 /**
+ * Encode and transmit PID
+ */
+static uint32_t blackboxWritePID(uint8_t pid)
+{
+    uint8_t kP, kI, kD;
+
+    if (IS_PID_CONTROLLER_FP_BASED(currentProfile->pidProfile.pidController)) {
+        if (pid < 3) {
+            kP = constrain(lrintf(currentProfile->pidProfile.P_f[pid] * 10.0f), 0, 250);
+            kI = constrain(lrintf(currentProfile->pidProfile.I_f[pid] * 100.0f), 0, 250);
+            kD = constrain(lrintf(currentProfile->pidProfile.D_f[pid] * 1000.0f), 0, 100);
+        }
+        else if (pid == PIDLEVEL) {
```

Comment:
I don't understand why you want to log PIDLEVEL as kP and kD etc? The level mode still uses rate PID's